src/foam/nanos/theme/ThemeDomain.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.theme',
+  name: 'ThemeDomain',
+
+  documentation: 'mapping of domain to Theme. Note: is managed by the ThemeDomainDAO',
+
+  tableColumns: [
+    'id',
+    'theme'
+  ],
+
+  properties: [
+    {
+      name: 'id',
+      class: 'String',
+      label: 'Domain'
+    },
+    {
+      name: 'theme',
+      class: 'Reference',
+      of: 'foam.nanos.theme.Theme',
+      tableCellFormatter: function(value, obj, axiom) {
+        this.__subSubContext__.themeDAO
+          .find(value)
+          .then((theme) => {
```

Comment:
```javascript
value$find.then((theme) => {
```
Will do the same.

Reply:
this was cut and paste, I don't know anything about u2 - unfortunately.

Reply:
upated.